flexget/components/sites/sites/alpharatio.py

```diff
@@ -295,14 +295,33 @@ def search(self, task, entry, config):
                 e['url'] = url
                 e['torrent_tags'] = torrent_tags
                 if not size:
-                    logger.error(
+                    logger.debug(
                         'No size found! Please create a Github issue. Size received: {}', size_col
                     )
                 else:
                     e['content_size'] = parse_filesize(size.group(0))
-                e['torrent_snatches'] = int(torrent_info[snatches_idx].text)
-                e['torrent_seeds'] = int(torrent_info[seeds_idx].text)
-                e['torrent_leeches'] = int(torrent_info[leeches_idx].text)
+
+                mappings_int = {
+                    'torrent_snatches': snatches_idx,
+                    'torrent_seeds': seeds_idx,
+                    'torrent_leeches': leeches_idx,
+                }
+
+                for mapping in mappings_int:
+                    dest = mapping
+                    src = mappings_int[mapping]
```

Comment:
The more pythonic way to do this would be with `.items()`
```suggestion
                for dest, src in mappings_int.items():
```

Reply:
Done! @frankyw if you can just double check if all is ok, can be merged